### ISSUE TITLE
Tooltips Are Completely Gray

### ISSUE THREAD

**Issue by TheVisionaryOne** [GenuineSounds/WorldTooltip#30]
For some reason, when I install this mod to my pack, all tooltips are completely gray. No matter how I mess with the config (colors, opacity, bypass colors, etc.), the tooltips still look the same.

![2020-05-21_21 03 22](https://user-images.githubusercontent.com/56095032/82718776-7bb98680-9c59-11ea-8090-61ac87524853.png)
![2020-05-22_18 09 45](https://user-images.githubusercontent.com/56095032/82718779-7d834a00-9c59-11ea-8f9e-95b838fb3e61.png)

[(Here's my mod list)](https://hastebin.com/jusohitixe.rb)

This mod looks awesome, and I'd really love to incorporate it into my modpack. This issue is most likely caused by an incompatibility with some other mod in my pack, but I don't know of any other mods that mess with tooltips. Do you have any ideas of what mod could possibly be causing this problem?

**Comment by PixelmonPRO** [comment]
> For some reason, when I install this mod to my pack, all tooltips are completely gray. No matter how I mess with the config (colors, opacity, bypass colors, etc.), the tooltips still look the same.
> 
> ![2020-05-21_21 03 22](https://user-images.githubusercontent.com/56095032/82718776-7bb98680-9c59-11ea-8090-61ac87524853.png)
> ![2020-05-22_18 09 45](https://user-images.githubusercontent.com/56095032/82718779-7d834a00-9c59-11ea-8f9e-95b838fb3e61.png)
> 
> [(Here's my mod list)](https://hastebin.com/jusohitixe.rb)
> 
> This mod looks awesome, and I'd really love to incorporate it into my modpack. This issue is most likely caused by an incompatibility with some other mod in my pack, but I don't know of any other mods that mess with tooltips. Do you have any ideas of what mod could possibly be causing this problem?

I have the same problem, but with a different set of mods. I tried to remove the mods, it came to the removal of all mods, the only way it could fix the problem: /

**Comment by axlmly** [comment]
i have the same problem,There is also a problem with neat modules.Sorry, I don't speak a very good foreign language.

**Comment by axlmly** [comment]
> > 出于某种原因，当我将此mod安装到我的包中时，所有工具提示都是灰色的。不管我如何弄乱配置（颜色，不透明度，旁路颜色等），工具提示看起来都一样。
> > ![2020-05-21_21 03 22](https://user-images.githubusercontent.com/56095032/82718776-7bb98680-9c59-11ea-8090-61ac87524853.png)
> > ![2020-05-22_18 09 45](https://user-images.githubusercontent.com/56095032/82718779-7d834a00-9c59-11ea-8f9e-95b838fb3e61.png)
> > [（这是我的mod列表）](https://hastebin.com/jusohitixe.rb)
> > 这个mod看起来很棒，我真的很想将它合并到我的modpack中。这个问题很可能是由于与我的包中的某些其他模块不兼容而引起的，但是我不知道其他任何与工具提示混淆的模块。您对可能导致此问题的mod有任何想法吗？
> 
> 我有同样的问题，但是有不同的mod。我试图删除mod，这涉及到删除所有mod，这是解决问题的唯一方法：/

Known problem mod:psi ,Better Burning

**Comment by TheVisionaryOne** [comment]
Yeah, the only solution I've found for this problem is to remove the mod(s) that World Tooltips is incompatible with. Better Burning was the mod that was clashing with it when I opened this issue. I've come across several more mods that are incompatible with World Tooltips as well, most recently including [Spiders 2.0](https://www.curseforge.com/minecraft/mc-mods/spiders-2-0).

**Comment by TheVisionaryOne** [comment]
Another update. I've done some testing, and I've found out this is only a soft incompatibility! If you ever experience this gray tooltips problem, just run your pack once without World Tooltips, and then put World Tooltips back in your mod folder. Should work as intended.
